portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -159,27 +160,25 @@ private String getUploadedFileUrl(HttpServletRequest request, String formInputEl
   }
   
   /** 
-   * Returns the score of the sentiment of the given message, 
+   * Returns the score of the sentiment of the given message,
    * which is a float from -1 to 1 representing how negative or positive the text it.
    */
   private float getSentimentScore(String message) throws IOException {
     Document doc =
         Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
     LanguageServiceClient languageService = LanguageServiceClient.create();
     Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
-    float score = sentiment.getScore();
     languageService.close();
-
-    return score;
+    return sentiment.getScore();
   }
 
   /** 
-   * Translates "originalText" to the language represented by "languageCode" and returns the translated text.
+   * Translates a piece of comment to the language represented by {@code languageCode}.
```

Comment:
You're translating the entire comment :)

```suggestion
   * Translates a comment to the language represented by {@code languageCode}.
```

Reply:
okay